### ISSUE TITLE
Prebid Server Adding an SChain node

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#1491]
We haven't actually been asked for this yet, but I predict that someday someone's going to want to have Prebid Server _add_ an schain node to an existing list in source.ext.schain.

Currently, PBS-Go can support _initializing_ the schain with the [default server request](https://docs.prebid.org/prebid-server/features/pbs-default-request.html), but neither version has the ability to add a node to the end of an existing list.

The proposal:
1. Add a new optional server-level config that lets the host company define an schain node.
2. If this config is specified and is valid:
   2a. If source.ext.schain doesn't exist, the configured schain node is added there as the first element in the node array
   2b. If source.ext.schain does exist, the configured schain node is appended to the end of the node array

e.g.
```
auction:
    host-schain-node: '{ "asi":"pbshostcompany.com", "sid":"00001", "rid":"BidRequest", "hp":1 }'
```
Would get added to the end of source.ext.schain.nodes:
```
  "source": {
    "ext": {
      "schain": {
        "ver":"1.0",
        "complete": 1,
        "nodes": [
          {
            "asi":"firstschainnode.com",
            "sid":"00001",
            "rid":"BidRequest1",
            "hp":1
          },
          { "asi":"pbshostcompany.com", "sid":"00001", "rid":"BidRequest", "hp":1 }
        ]     
      } 
    }
  }
```

**Comment by SyntaxNode** [comment]
> We haven't actually been asked for this yet,

Shouldn't we wait until we discover a need to implement such a feature?

**Comment by AlexBVolcy** [comment]
This issue is now being worked on by me! I expect a PR to be out early next week.

**Comment by AlexBVolcy** [comment]
This feature has been implemented and merged in, and will be released in PBS-Go v0.217.0
